README.md

```diff
@@ -181,29 +181,52 @@ it into the "Index pattern" field, then click the "Next step" button.
 Discover section.
 
 
-#### Using Docker
+#### Using Docker Locally
 
-1. Download, install, and launch [Docker]
+1. Download, install, and launch [Docker](https://www.docker.com/products/docker-desktop)
 
-1. Set up the Docker image
+1. Build and launch the Docker containers: `docker-compose up`
 
-  ```
-  $ bin/setup --docker
-  ```
+1. Copy necessary config files locally. This dir is mounted as a volume, so new files also appear in the container.
+
+```sh
+cp config/application.yml.default config/application.yml
+cp -R certs.example certs
+cp -R keys.example keys
+cp pwned_passwords/pwned_passwords.txt.sample pwned_passwords/pwned_passwords.txt
+cp config/service_providers.localdev.yml config/service_providers.yml
+```
+
+1. Manually edit `application.yml`, in the `development:` section to make Redis look like
```

Comment:
Ok so it looks like with the right environment variable set, we can override the `application.yml`, so if there's a way to set environment variables inside the docker container, we can automate this bit

Here's Figaro checking for the ENV variable already existing it and skipping it so
https://github.com/laserlemon/figaro/blob/master/lib/figaro/application.rb#L79-L81

Reply:
Demo:

```
$ export redis_url="AAAAAA"
$ make run
# ... snip ...
15:39:31 web.1         | WARNING: Skipping key "redis_url". Already set in ENV.
```

and then the app blows up because I gave it a garbage redis URL

Reply:
🤔 I wonder if it's just that it's `REDIS_URL` in docker-compose and `redis_url` elsewhere.  Will test that out

Reply:
yep `web_1          | WARNING: Skipping key "redis_url". Already set in ENV.`